pkg/auth/handler/webapp/authflow_controller.go

```diff
@@ -152,7 +153,10 @@ func (c *AuthflowController) HandleStartOfFlow(
 	if err != nil {
 		if errors.Is(err, authflow.ErrFlowNotFound) {
 			screen, err := c.createScreen(r, s, flowReference, input)
-			if err != nil {
+			if errors.Is(err, declarative.ErrNoPublicSignup) {
+				c.renderError(w, r, err)
```

Comment:
Should update makeErrorResult to explicitly mark ErrNoPublicSignup as non recoverable error?